scene/animation/easing_equations.h

```diff
@@ -402,4 +402,33 @@ static real_t out_in(real_t t, real_t b, real_t c, real_t d) {
 }
 }; // namespace back
 
+namespace spring {
+static real_t out(real_t t, real_t b, real_t c, real_t d) {
+	t /= d;
+	float s = 1.0 - t;
```

Comment:
```suggestion
	real_t s = 1.0 - t;
```

Reply:
are you sure i should change that one float to a real?
i was thinking about changing the two reals to floats because tho all parameters are real all variables in file are float.
so this is inconsistent?

Reply:
If you want to use a float here, you should write it to `1.0f - t`. Well, since Godot uses double values for delta values for process and animation elapsed time, so it would basically be preferable to have it be `real_t`. There are inconsistencies in some easing, so maybe that could be revised later as a separate PR.